appveyor.yml

```diff
@@ -3,10 +3,12 @@ platform: x86
 clone_folder: c:\projects\fos-message
 
 cache:
-  - '%LOCALAPPDATA%\Composer\files'
```

Comment:
I'm not sure it's worth it according to nicolas: https://github.com/puli/issues/issues/169#issuecomment-169315809. What's your opinion on this?